pages/instructorAssessmentGroups/instructorAssessmentGroups.js

```diff
@@ -0,0 +1,138 @@
+const ERR = require('async-stacktrace');
+const express = require('express');
+const router = express.Router();
+const path = require('path');
+const debug = require('debug')('prairielearn:' + path.basename(__filename, '.js'));
+
+const error = require('@prairielearn/prairielib/error');
+const groupUpdate = require('../../lib/group-update');
+const sqldb = require('@prairielearn/prairielib/sql-db');
+const sqlLoader = require('@prairielearn/prairielib/sql-loader');
+
+const sql = sqlLoader.loadSqlEquiv(__filename);
+
+router.get('/', function(req, res, next) {
+    debug('GET /');
+    const params = {assessment_id: res.locals.assessment.id};
+    sqldb.query(sql.config_info, params, function(err, result) {
+        if (ERR(err, next)) return;
+        res.locals.config_info = result.rows[0];
+        sqldb.query(sql.assessment_list, params, function(err, result) {
+            if (ERR(err, next)) return;
+            res.locals.assessment_list_rows = result.rows;
+            sqldb.query(sql.select_groups, params, function(err, result) {
+                if (ERR(err, next)) return;
+                res.locals.groups_rows = result.rows;
+                sqldb.query(sql.not_assigned_users, params, function(err, result) {
+                    if (ERR(err, next)) return;
+                    res.locals.not_assigned_users_rows = result.rows;
+                    debug('render page');
+                    res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+                });
+            });
+        });
+    });
```

Comment:
Maybe consider refactoring this to use [`async.series()`](https://caolan.github.io/async/v3/docs.html#series)?